examples/Bickley_jet_shallow_water.jl

```diff
@@ -0,0 +1,247 @@
+
+set!(model, uh = uhⁱ , h = hⁱ)
+
+# We compute the total vorticity and the perturbation vorticity.
+
+uh, vh, h = model.solution
+        u = ComputedField(uh / h)
+        v = ComputedField(vh / h)
+  ω_field = ComputedField( ∂x(vh/h) - ∂y(uh/h) )
+   ω_pert = ComputedField( ω_field - Ω )
+
+# Progress will output the clock times and the norm of the cross channel velocity,
+# as this is used to compute the growth rates.  We obtain the `norm` function from `LinearAlgebra`.
+
+using LinearAlgebra, Printf
+
+function progress(sim)
+    compute!(v)
+    @printf("Iteration: %d, time: %s, norm v: %f\n",
+    sim.model.clock.iteration,
+    prettytime(sim.model.clock.time),
+    norm(interior(v)) )
+end
+nothing # hide
+
+# ## Running a `Simulation`
+#
+# We pick the time-step that ensures to resolve the surface gravity waves.
+# A time-step wizard can be applied to use an adaptive time step.
+
+simulation = Simulation(model, Δt = 1e-2, stop_time = 150.0, progress=progress)
+
+# ## Prepare output files
+#
+# Define a function to compute the growth rate based on the cross channel velocity
+
+function growth_rate(model)
+    compute!(v)
+    return norm(interior(v))
+end
+nothing # hide
+
+# Choose the two fields to be output to be the total and perturbation vorticity.
+
+outputs = (ω_total = ω_field, ω_pert = ω_pert)
+
+# Build the `output_writer` for the two-dimensional fields to be output.
+# Output every `t = 1.0`.
+
+simulation.output_writers[:fields] =
+    NetCDFOutputWriter(
+        model,
+        (ω = ω_field, ωp = ω_pert),
+        filepath=joinpath(@__DIR__, "Bickley_jet_shallow_water.nc"),
+        schedule=TimeInterval(1.0),
+        mode = "c")
+
+# Build the `output_writer` for the growth rate, which is a scalar field.
+# Output every time step.
+#
+
+simulation.output_writers[:growth] =
+    NetCDFOutputWriter(
+        model,
+        (growth_rate = growth_rate,),
+        filepath="growth_rate_shallow_water.nc",
+        schedule=IterationInterval(1),
+        dimensions=(growth_rate=(),),
+        mode = "c")
+
+run!(simulation)
+
+# ## Visiualize the results
+#
+
+using NCDatasets, Plots, IJulia
+nothing # hide
+
+# Define the coordinates for plotting
+
+xf = xnodes(ω_field)
+yf = ynodes(ω_field)
+nothing # hide
+
+# Define keyword arguments for plotting the contours
+
+kwargs = (
+         xlabel = "x",
+         ylabel = "y",
+           fill = true,
+         levels = 20,
+      linewidth = 0,
+          color = :balance,
+       colorbar = true,
+           ylim = (0, Ly),
+           xlim = (0, Lx)
+)
+nothing # hide
+
+# Read in the `output_writer` for the two-dimensional fields
+# and then create an animation showing both the total and perturbation
+# vorticities.
+
+ds = NCDataset(simulation.output_writers[:fields].filepath, "r")
+
+iterations = keys(ds["time"])
+
+anim = @animate for (iter, t) in enumerate(ds["time"])
+        ω = ds["ω"][:,:,1,iter]
+    ωp = ds["ωp"][:,:,1,iter]
```

Comment:
spaces after commas

Reply:
Happy to follow this convention, as it is easier to read